### PR TITLE
updated case sensitivity in table.sql parsing.

### PR DESCRIPTION
**Pull request by marcoooo**
Added for safety net disabling keys / enabling keys when loading data from txt file.

### CODE REVIEW

File: src/python/ensembl/database/unittestdb.py [hunk 1]
```diff
@@ -95,8 +99,14 @@ def __init__(self, url: URL, dump_dir: os.PathLike, name: Optional[str] = None)
...
                             conn.execute(f"DELETE FROM {table}")
+
                         self._load_data(conn, table, filepath)
-        except:
+                if self.dbc.dialect == 'mysql':
+                    conn.execute(f"SET FOREIGN_KEY_CHECKS=1;")
+                elif self.dbc.dialect == 'sqlite':
+                    conn.execute(f"PRAGMA foreign_keys = ON;")
+        except Exception as exc:
+            print(exc)
```
Comment (JAlvarezJarreta) [inline src/python/ensembl/database/unittestdb.py:109]:
```suggestion
        except:
```
The exception is raised after dropping the database, so I believe there is no need to catch it and print it.

**Review by JAlvarezJarreta — COMMENTED** [review]
Nice additions, just a single comment to address before approval :+1:

File: src/python/ensembl/database/unittestdb.py (hunk 1 above)
Comment (marcoooo) [reply src/python/ensembl/database/unittestdb.py]: Let's call this a typo :-D

**Review by JAlvarezJarreta — CHANGES_REQUESTED** [review]

File: src/python/ensembl/database/unittestdb.py [hunk 2]
```diff
@@ -29,17 +29,16 @@
...
 import subprocess
+from pathlib import Path
 from typing import Iterator, Optional, Union
 
 import sqlalchemy
 from sqlalchemy import create_engine, text
 from sqlalchemy.engine.url import make_url
-from sqlalchemy_utils import database_exists
-
+from sqlalchemy import exc
```
[the remark this answers is not in the document]
Comment (marcoooo) [reply src/python/ensembl/database/unittestdb.py:41]: I don't believe I added it with this PR, but sure.

File: src/python/ensembl/database/unittestdb.py [hunk 3]
```diff
@@ -95,11 +98,16 @@ def __init__(self, url: URL, dump_dir: os.PathLike, name: Optional[str] = None)
...
                         else:
                             conn.execute(f"DELETE FROM {table}")
+
                         self._load_data(conn, table, filepath)
-        except:
+                if self.dbc.dialect == 'mysql':
+                    conn.execute(f"SET FOREIGN_KEY_CHECKS=1;")
+                elif self.dbc.dialect == 'sqlite':
+                    conn.execute(f"PRAGMA foreign_keys = ON;")
+        except exc.DBAPIError as e:
```
[the remark this answers is not in the document]
Comment (marcoooo) [reply src/python/ensembl/database/unittestdb.py]: I updated with the highest possible in SQLAlchemy. I tend not to like the too broad error.... a tests script parameter error shouldn't drop the db for instance.

File: src/python/ensembl/database/unittestdb.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (marcoooo) [reply src/python/ensembl/database/unittestdb.py:41]: Removed from deps

**Review by JAlvarezJarreta — APPROVED** [review]

**Pull request merged**